fast_plotter/postproc/functions.py

```diff
@@ -51,13 +51,13 @@ def handle_one_df(df, query=None, replacements=[],
     return df
 
 
-def query(df, query):
+def query(df, query, engine_query=None):
```

Comment:
Can we call this just `engine`?
```suggestion
def query(df, query, engine=None):
```